packages/rn-tester/e2e/test/features/buttonComponentScreen.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Button component screen
+
+    Scenario: Cancel Button
+        Given User is on the main screen
+        Then Verify that the Button component is displayed
+        When User clicks on the Button component
+        Then Verify that the "Button" header is displayed
+        When User clicks on the Cancel Application button
+        Then Verify that the alert box has text: "Your application has been cancelled!"
+        When User clicks on the OK button
```

Comment:
I would add here one more _Then_ step e.g. 
`Then User is on the main screen`

Reply:
fixed
`Then Verify that the "Button" header is displayed` has been added